### PR TITLE
Error message based on file size restriction.

### PR DESCRIPTION
**Pull request by jwillmer**
As described in #2: Displaying the error message file size limit in MB if the limit exceeds 1MB.

### CODE REVIEW

File: juicy-filedrop.html [hunk 1]
```diff
@@ -83,12 +83,22 @@
...
                 }
                 if (file.size > this.maxfilesize) {
+                    var maxfilesizeKb = this.maxfilesize / 1024,
+                        maxfilesizeMb = maxfilesizeKb / 1024,
+                        maxfilesizeString = '';
+
+                    if (maxfilesizeKb > 1024) {
+                        maxfilesizeString = '' + parseFloat(maxfilesizeMb).toFixed(2) + 'MB';
+                    } else {
+                        maxfilesizeString = '' + parseFloat(maxfilesizeKb).toFixed(2) + 'KB';
```
Comment (warpech) [inline juicy-filedrop.html]: I think that `parseFloat` is redundant here, because both variables are actually numbers already. Do you agree?

**Comment by warpech** [comment]
Looking good, just one comment. Can you pls take a look?

**Comment by jwillmer** [comment]
I agree, it is redundant. I have removed it.

**Comment by warpech** [comment]
Great! I will merge and release a new version

**Comment by warpech** [comment]
Merged. I will issue a new version tomorrow.

**Review by warpech — COMMENTED** [review]
Job well done! 👍

**Pull request merged**